src/tailwind.css

```diff
@@ -1,3 +1,3 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+@tailwind utilities;
```

Comment:
@jakedcolbert Did you mean to change src/tailwind.css? It looks like you deleted the trailing newline from the file and prettier doesn't like this.  To fix this:

1. run `npm run format`
2. check in the changes.

Reply:
That was a mistake, I'll make a habit of running `npm run format` at each commit. Thanks